core/client/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -120,11 +122,11 @@ public static FileInStream create(URIStatus status, InStreamOptions options) {
    * @param status the file status
    * @param options the client options
    */
-  protected FileInStream(URIStatus status, InStreamOptions options) {
+  protected FileInStream(URIStatus status, InStreamOptions options, FileSystemContext context) {
```

Comment:
In another PR we put the context first, do we want to have a consistent argument ordering?